app/models/Property.ts

```diff
@@ -63,7 +69,7 @@ export class Property {
     this.postcode = params.postcode;
     this.houseType = params.houseType;
     this.numberOfBedrooms = params.numberOfBedrooms;
-    this.age = params.age;
+    this.age = params.age - 1; // Subtract 1 because years should be indexed to 0
```

Comment:
Helpful comment! This feels a bit tricky though as "age" indicates a non-zero indexed figure.

Should we just return `newBuildPrice` if the age is 0 or 1?

Reply:
Yep, that's what's going on
```    
if (this.age === 0) return this.newBuildPrice; // If newbuild, return newBuildPrice and don't depreciate
```

I think it got a bit confusing because we are using 1 as a stand-in for newbuilds in the 'age' form field (I have now made a note to change that), whereas some of the lifetime forecasting math wanted `y = 0` for the first year (eg if appreciating by something like `inflatedPrice = (1 + inflation) ^ y`).

Does that make sense?? Not sure if I've explained myself well here!